internal/server/recon/find_entities.go

```diff
@@ -166,10 +166,17 @@ func resolvePlaceIDs(
 				if err != nil {
 					return err
 				}
-				// Only keep the first place ID, as the rest ones are usually much less accurate.
-				resolveResultChan <- resolveResult{
-					entityInfo: &entityInfo,
-					placeIDs:   []string{placeIDs[0]}}
+				if len(placeIDs) == 0 {
```

Comment:
nit:
`
usedPlaceIds = []string{}
if len(placeIDs) > 0 {
usedPlaceds = []string{placeIDs[0]}
}
resolveResultChan <- resolveResult{..., placeIDs: usedPlaceids}

Reply:
This is much more elegant. Fixed!